js/converter.js

```diff
@@ -49,7 +53,52 @@ function convertPoints() {
 	if (document.getElementById('EPSG4326').checked != true) {
 		geojson = convertEPSG3857to4326(geojson);
 	}
-    geoJsonToOutput(geojson);
+
+	return geojson;
+}
+// convertPairs works on pairs of latitudes and longitues to create a polygon.
+// Made to handle cases where people send a request like:
+//   LAT between -9.646536, 6.998524, LON between 94.51485, 140.9909
+function convertPairs() {
+	let lats = document.getElementsByName("lats")[0].value.split(","),
+		lons = document.getElementsByName("lons")[0].value.split(",");
+
+	if(lats.length !== lons.length) {
+		alert(
+			"not the equal number of latitudes("+ lats.length
+			+") and longitudes("+ lons.length +")");
+	}
+
+	let coords = [];
+	for(const lat of lats) {
+		for(const lon of lons) {
+			let ll = parseInput(lat, lon)
+			coords.push([ll.lon, ll.lat])
+		}
+	}
+	if (coords.length < 4) {
+		alert("Warning: polygons require at least 4 points.")
+	}
+	coords = normalizePolygon(coords);
+	coords.push(coords[0]) // Close polygon by looping back to itself
+
+	geoJsonToOutput(toGeoJSON(coords));
+}
+// Normalizes the order of changes in a polygon
+// Essentially, when you are making a polygon if you don't traverse
+// the changes in order you end up with coordinates crossing themselves,
+// turning them into a hour-glass shape - X.
+function normalizePolygon(coords) {
+	// A hard-coded fix for a box which is traversed by doing all lats and
+	// then all lons, by swapping 3/4 we'll get the changes in sequence
+	// and it'll look good
+	if (coords.length === 4) {
+		const third = coords[2];
+		coords[2] = coords[3];
+		coords[3] = third;
+	}
```

Comment:
this is definitely a bit odd. But, without this, you get things like:

<img width="181" alt="GeoJSON_Conversion_Tool" src="https://user-images.githubusercontent.com/18167/81386309-63b91180-9147-11ea-81dc-8748cef2308c.png">

I'm certain there is a proper algorithm for how to lay out the coordinates to avoid this. But I don't know what it is and I don't feel like spending more time than this to fix it :P 

You could benefit from having this in the `convertPoints` function as well, but I didn't want to introduce the change without discussing it.

Reply:
I think the algorithm should be something like interweaving the lat/lon pairs, I'll mull it over for a bit in case I can think of something better :)